README.adoc

```diff
@@ -193,7 +192,7 @@ Create a `start/src/main/java/io/openliberty/guides/config/CustomEmailConverter.
 include::finish/src/main/java/io/openliberty/guides/config/CustomEmailConverter.java[tags=customConfig]
 ----
 
-To register your implementation, create a `start/src/main/resources/META-INF/services/org.eclipse.microprofile.config.spi.Converter` file and add the fully qualified class name of the custom converter into it:
+To register your implementation, create a file called `org.eclipse.microprofile.config.spi.Converter` under `start/src/main/resources/META-INF/services/` directory and add the fully qualified class name of the custom converter into it:
```

Comment:
We use "fully qualified class name" here but "full class name" above.  Should they be the same?